tsdb/agent/db.go

```diff
@@ -335,7 +335,7 @@ func validateOptions(opts *Options) *Options {
 		opts.WALCompression = wlog.CompressionNone
 	}
 
-	// Revert Stripesize to DefaultStripsize if Stripsize is either 0 or not a power of 2.
+	// Revert StripeSize to DefaultStripSize if StripSize is either 0 or not a power of 2.
```

Comment:
I think the other two `{Default}StripSize` mentions here should be `{Default}StripeSize` too.

Reply:
Done.